### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by harjas27**
**What type of PR is this?**

/kind bug
/sig cli

**What this PR does / why we need it**:
Fixes a typo in one of the enumerations. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/polymorphichelpers/rollout_status.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func (s *DaemonSetStatusViewer) Status(obj runtime.Unstructured, revision int64)
 	}
 
 	if daemon.Spec.UpdateStrategy.Type != appsv1.RollingUpdateDaemonSetStrategyType {
-		return "", true, fmt.Errorf("rollout status is only available for %s strategy type", appsv1.RollingUpdateStatefulSetStrategyType)
```
Comment (SergeyKanzhelev) [inline staging/src/k8s.io/kubectl/pkg/polymorphichelpers/rollout_status.go:105]: just a small improvement suggestion - output `daemon.Spec.UpdateStrategy.Type` as an additional piece of information for easier troubleshooting.

Anyway, it is clearly a copy/paste error that can confuse,

/lgtm

**Review by soltysh — APPROVED** [review]
/approve

**Comment by SergeyKanzhelev** [comment]
@harjas27 please sign CLA
/release-notes-none
/priority backlog
/triage accepted

**Comment by harjas27** [comment]
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.

I have signed the CLA

**Pull request closed without merging**